.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ workflows:
       - architect/go-build:
           name: go-build
           binary: aws-operator
-          resource_class: large
+          resource_class: xlarge
```

Comment:
I have seen many errors building aws-operator  since the process was killed due low resources so bumping.